handbook/best-practice/system-admins.md

```diff
@@ -0,0 +1,33 @@
+# System Admins
+
+The System Admins for each of the systems DVELP uses are defined in the below
+table. It is the responsibility of the System Admin to make sure that only the relevant
+people have access to these systems and at the right permission level.
+
+We operate on the principle of least privilege, which means individuals should have
+the minimum level of privilege in order to be able to perform their role
+effectively.
+
+System Admins conduct a thorough review of the permissions once every calendar
+month and adjust permissions ad hoc whenever required due to a person
+joining/leaving the team and/or a project starting/ending.
+
+Deputy System Admins are required to have admin permissions on the relevant
+platforms. This is to enable them to handle urgent requests while System Admins
+are absent. Deputy System Admins are not required to conduct the thorough
+monthly review. 
+
+| System | System Admin | Deputy System Admin |
+| --- | --- | --- |
+| CharlieHR | Andreas Kater | Ruby Williams |
+| Forecast  | Ross Wilson | Annise Makki |
+| Github | Ilya Lozer | Vitalii Prodan |
+| GSuite | Ruby Williams | Andreas Kater |
+| Harvest | Ross Wilson | Annise Makki |
+| Heroku | Ross Wilson | Annise Makki |
+| Looker | Vitalii Prodan | Ilya Lozer |
+| Slack | Ruby Williams | Andreas Kater |
+| Trello | Annise Makki | Ross Wilson |
+| Twilio | Vitalii Prodan | Ilya Lozer |
+| Redshift | Tom Mullen | Vitalii Prodan |
```

Comment:
@andreas-kater can we have them in alpha order?

Reply:
done